lms/djangoapps/course_structure_api/v0/views.py

```diff
@@ -179,9 +205,27 @@ class CourseStructure(CourseViewMixin, RetrieveAPIView):
 
     **Response Values**
 
-        * root: ID of the root node of the structure
+        * root: The ID of the root node of the course structure.
+
+        * blocks: A dictionary that maps block IDs to a collection of
+          information about each block. Each block contains the following
+          fields:
+
+          * id: The ID of the block.
+
+          * type: The type of block. Possible values include sequential,
+            vertical, html, problem, video, and discussion. The type can also be
+            the name of a custom type of block used for the course.
+
+          * display_name: The display name configured for the block.
+
+          * graded: Whether or not the sequential or problem is graded. true or
+            false.
+
+          * format: ??
```

Comment:
`format` is the `assignment_type` (from the grading policies endpoint) that this block contributes to.